agent/nmap_agent.py

```diff
@@ -13,6 +13,7 @@
 from ostorlab.agent.message import message as msg
 from ostorlab.agent.mixins import agent_persist_mixin as persist_mixin
 from ostorlab.agent.mixins import agent_report_vulnerability_mixin as vuln_mixin
+from ostorlab.assets import ipv4, ipv6, domain_name
```

Comment:
I think it would be cleaner to use domain_name_asset and keep the variable name as is.